packages/graphback-codegen-client/src/templates/gqlTemplates.ts

```diff
@@ -21,12 +21,12 @@ ${returnFieldsString}
 } `
 }
 
-export const findOneQuery = (t: GraphQLObjectType) => {
-  const fieldName = getFieldName(t.name, GraphbackOperationType.FIND_ONE)
+export const findOneQuery = (t: ModelDefinition) => {
+  const fieldName = getFieldName(t.graphqlType.name, GraphbackOperationType.FIND_ONE)
 
-  return `query ${fieldName}($id: ID!) {
```

Comment:
Was there a decision made whether we are sticking to `id: ID` for the query? Wondering if this maybe deserves a 3-way call to discuss.

Reply:
We have the conversation going on https://github.com/aerogear/graphback/pull/1769#discussion_r462110979, but we have not reached a decision yet. 

/cc @wtrocki

Reply:
Decision reached. Current implementation works for me (simplicity + scalar support)